app/views/internal/feedback_messages/index.html.erb

```diff
@@ -108,6 +108,25 @@
   </div>
 </div>
 
+<div class="row m-3 justify-content-end">
```

Comment:
I think instead of having the filters at the top in this form factor, having buttons next to the reporter and the reported url that allow to "filter by this" instance. Possibly conditional to whether `@feedback_messages` contains that reporter or URL would make sense?

I think the controller can stay the same and moving the filtering into the report card could be ideal.

We _could_ also keep the filter top filter area if you think that is good to have as well, and in that case I think the elements should be inlined to tighten up that area in terms of vertical space. But I think shifting this part of the UI to the above proposal and then making the filtering UI a different PR makes sense because I have some thoughts that I'll leave some clarifying thoughts in #5012 which could help sew this up.

Reply:
Ideally the filters would actually be in a different part of the UI altogether, but I think because there are so many contact areas on this one surface it's hard to group them effectively.

I'll inline them for now, if you think that is a better way to organize this view.

Reply:
See what I just dropped in #5012 I think the big cut we need which is easier to reason about is "sorting" which effectively gives us what we want in terms of "grouping" by these things in a really glanceable way.